swayfmt/src/utils/map/comments.rs

```diff
@@ -193,9 +193,42 @@ fn get_comments_between_spans(
                 // There is a comment before this one, so we should get the context starting from the last comment's end to the beginning of the current comment
                 comments_with_context[index - 1].0.span.end()
             };
+
+            let mut rest_of_code = unformatted_code
+                .get(comment_span.end..)
+                .unwrap_or_default()
+                .lines()
+                .take(2);
+
+            // consume '\n'
+            let _ = rest_of_code.next();
```

Comment:
Hmm, this works too, I chose to use the first variation to make it clearer that we're not interested in the first line (which is a newline), and I think the semantics with that might be clearer, instead of skipping it over in the same line.